changelog.d/20250404_172727_30907815+rjmello_ha_endpoint_id.rst

```diff
@@ -0,0 +1,11 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Enabled support for registering a function to a specific HA endpoint via the
+  ``ha_endpoint_id`` argument in the following methods:
+
+  - ``Executor.register_function()``
+  - ``Client.register_function()``
```

Comment:
Consider linking these via the `:func:~...` syntax.  Examples of how to hook it up more beautifully at bottom of `changelog.rst`.  Will link to the methods and be monospaced.

Reply:
I don't see this directive used previously in `changelog.rst`, but good idea regardless.